+ source ../common/common_lib.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Sat Sep  2 00:44:08 2023 - INFO  - Start environmental preparation.
+ DNF_INSTALL strongswan
+ pkgs=strongswan
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs strongswan --node 1 --tempfile ''
+ tmpfile2=/tmp/tmp404ezz35
+ '[' -z '' ']'
+ tmpfile=/tmp/tmp404ezz35
+ touch /etc/strongswan/ipsec.d/triplets.dat /etc/tnc_config
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Sat Sep  2 00:45:35 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Sat Sep  2 00:45:37 2023 - INFO  - Start testing...
+ test_execution strongswan-swanctl.service
+ service=strongswan-swanctl.service
++ date '+%Y-%m-%d %T'
+ log_time='2023-09-02 00:45:37'
+ test_restart strongswan-swanctl.service
+ service=strongswan-swanctl.service
+ systemctl restart strongswan-swanctl.service
Failed to restart strongswan-swanctl.service: Unit strongswan-swanctl.service not found.
+ CHECK_RESULT 5 0 0 'strongswan-swanctl.service restart failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='strongswan-swanctl.service restart failed'
+ exit_mode=0
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'strongswan-swanctl.service restart failed'
+ LOG_ERROR 'strongswan-swanctl.service restart failed'
+ message='strongswan-swanctl.service restart failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'strongswan-swanctl.service restart failed'
Sat Sep  2 00:45:38 2023 - ERROR - strongswan-swanctl.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Sat Sep  2 00:45:39 2023 - ERROR - ../common/common_lib.sh line 34
+ '[' 0 -eq 1 ']'
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status strongswan-swanctl.service
+ grep 'Active: active'
Unit strongswan-swanctl.service could not be found.
+ CHECK_RESULT 1 0 0 'strongswan-swanctl.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='strongswan-swanctl.service restart failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'strongswan-swanctl.service restart failed'
+ LOG_ERROR 'strongswan-swanctl.service restart failed'
+ message='strongswan-swanctl.service restart failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'strongswan-swanctl.service restart failed'
Sat Sep  2 00:45:48 2023 - ERROR - strongswan-swanctl.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Sat Sep  2 00:45:49 2023 - ERROR - ../common/common_lib.sh line 37
+ '[' 0 -eq 1 ']'
+ return 0
+ systemctl stop strongswan-swanctl.service
Failed to stop strongswan-swanctl.service: Unit strongswan-swanctl.service not loaded.
+ CHECK_RESULT 5 0 0 'strongswan-swanctl.service stop failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='strongswan-swanctl.service stop failed'
+ exit_mode=0
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'strongswan-swanctl.service stop failed'
+ LOG_ERROR 'strongswan-swanctl.service stop failed'
+ message='strongswan-swanctl.service stop failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'strongswan-swanctl.service stop failed'
Sat Sep  2 00:45:50 2023 - ERROR - strongswan-swanctl.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 39'
+ message='../common/common_lib.sh line 39'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 39'
Sat Sep  2 00:45:52 2023 - ERROR - ../common/common_lib.sh line 39
+ '[' 0 -eq 1 ']'
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status strongswan-swanctl.service
+ grep 'Active: inactive'
Unit strongswan-swanctl.service could not be found.
+ CHECK_RESULT 1 0 0 'strongswan-swanctl.service stop failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='strongswan-swanctl.service stop failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'strongswan-swanctl.service stop failed'
+ LOG_ERROR 'strongswan-swanctl.service stop failed'
+ message='strongswan-swanctl.service stop failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'strongswan-swanctl.service stop failed'
Sat Sep  2 00:46:00 2023 - ERROR - strongswan-swanctl.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 42'
+ message='../common/common_lib.sh line 42'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 42'
Sat Sep  2 00:46:01 2023 - ERROR - ../common/common_lib.sh line 42
+ '[' 0 -eq 1 ']'
+ return 0
+ systemctl start strongswan-swanctl.service
Failed to start strongswan-swanctl.service: Unit strongswan-swanctl.service not found.
+ CHECK_RESULT 5 0 0 'strongswan-swanctl.service start failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='strongswan-swanctl.service start failed'
+ exit_mode=0
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'strongswan-swanctl.service start failed'
+ LOG_ERROR 'strongswan-swanctl.service start failed'
+ message='strongswan-swanctl.service start failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'strongswan-swanctl.service start failed'
Sat Sep  2 00:46:02 2023 - ERROR - strongswan-swanctl.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Sat Sep  2 00:46:04 2023 - ERROR - ../common/common_lib.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ grep 'Active: active'
+ systemctl status strongswan-swanctl.service
Unit strongswan-swanctl.service could not be found.
+ CHECK_RESULT 1 0 0 'strongswan-swanctl.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='strongswan-swanctl.service start failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'strongswan-swanctl.service start failed'
+ LOG_ERROR 'strongswan-swanctl.service start failed'
+ message='strongswan-swanctl.service start failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'strongswan-swanctl.service start failed'
Sat Sep  2 00:46:12 2023 - ERROR - strongswan-swanctl.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Sat Sep  2 00:46:13 2023 - ERROR - ../common/common_lib.sh line 47
+ '[' 0 -eq 1 ']'
+ return 0
+ test_enabled strongswan-swanctl.service
+ service=strongswan-swanctl.service
++ systemctl is-enabled strongswan-swanctl.service
+ state=not-found
+ '[' not-found == enabled ']'
+ '[' not-found == disabled ']'
+ '[' not-found == masked ']'
+ '[' not-found == static ']'
+ LOG_INFO 'Unit is indirect, ignoring.'
+ message='Unit is indirect, ignoring.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Unit is indirect, ignoring.'
Sat Sep  2 00:46:15 2023 - INFO  - Unit is indirect, ignoring.
+ journalctl --since '2023-09-02 00:45:37' -u strongswan-swanctl.service
+ grep -i 'fail\|error'
+ grep -v -i 'DEBUG\|INFO\|WARNING'
+ CHECK_RESULT 1 0 1 'There is an error message for the log of strongswan-swanctl.service'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of strongswan-swanctl.service'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ systemctl start strongswan-swanctl.service
Failed to start strongswan-swanctl.service: Unit strongswan-swanctl.service not found.
+ sed -i 's\ExecStart=/usr/sbin/charon-systemd\ExecStart=/usr/sbin/charon-systemd -v\g' /usr/lib/systemd/system/strongswan-swanctl.service
sed: can't read /usr/lib/systemd/system/strongswan-swanctl.service: No such file or directory
+ systemctl daemon-reload
+ systemctl reload strongswan-swanctl.service
Failed to reload strongswan-swanctl.service: Unit strongswan-swanctl.service not found.
+ CHECK_RESULT 5 0 0 'strongswan-swanctl.service reload failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='strongswan-swanctl.service reload failed'
+ exit_mode=0
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'strongswan-swanctl.service reload failed'
+ LOG_ERROR 'strongswan-swanctl.service reload failed'
+ message='strongswan-swanctl.service reload failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'strongswan-swanctl.service reload failed'
Sat Sep  2 00:46:22 2023 - ERROR - strongswan-swanctl.service reload failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_strongswan-swanctl.sh line 36'
+ message='oe_test_service_strongswan-swanctl.sh line 36'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_strongswan-swanctl.sh line 36'
Sat Sep  2 00:46:23 2023 - ERROR - oe_test_service_strongswan-swanctl.sh line 36
+ '[' 0 -eq 1 ']'
+ return 0
+ systemctl status strongswan-swanctl.service
+ grep 'Active: active'
Unit strongswan-swanctl.service could not be found.
+ CHECK_RESULT 1 0 0 'strongswan-swanctl.service reload causes the service status to change'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='strongswan-swanctl.service reload causes the service status to change'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'strongswan-swanctl.service reload causes the service status to change'
+ LOG_ERROR 'strongswan-swanctl.service reload causes the service status to change'
+ message='strongswan-swanctl.service reload causes the service status to change'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'strongswan-swanctl.service reload causes the service status to change'
Sat Sep  2 00:46:25 2023 - ERROR - strongswan-swanctl.service reload causes the service status to change
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_strongswan-swanctl.sh line 38'
+ message='oe_test_service_strongswan-swanctl.sh line 38'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_strongswan-swanctl.sh line 38'
Sat Sep  2 00:46:26 2023 - ERROR - oe_test_service_strongswan-swanctl.sh line 38
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Sat Sep  2 00:46:28 2023 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 8
+ test 8 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Sat Sep  2 00:46:29 2023 - ERROR - failed to execute the case.
+ exit 8
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Sat Sep  2 00:46:30 2023 - INFO  - start environment cleanup.
+ sed -i 's\ExecStart=/usr/sbin/charon-systemd -v\ExecStart=/usr/sbin/charon-systemd\g' /usr/lib/systemd/system/strongswan-swanctl.service
sed: can't read /usr/lib/systemd/system/strongswan-swanctl.service: No such file or directory
+ systemctl daemon-reload
+ systemctl reload strongswan-swanctl.service
Failed to reload strongswan-swanctl.service: Unit strongswan-swanctl.service not found.
+ rm -rf /etc/strongswan/ipsec.d/triplets.dat /etc/tnc_config
+ systemctl stop strongswan-swanctl.service
Failed to stop strongswan-swanctl.service: Unit strongswan-swanctl.service not loaded.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmp404ezz35 ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmp404ezz35
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Sat Sep  2 00:47:06 2023 - INFO  - Finish environment cleanup!
